modules/ding_gatewayf/ding_gatewayf.info

```diff
@@ -0,0 +1,6 @@
+name = Ding gatewayf
+description = Support for login at gatewayf.
+core = 7.x
+package = Ding!
+dependencies[] = ding_provider
```

Comment:
How about also adding a `hook_ding_provider_user()` implementation? As I see it this module depends on a wayf compatible provider.